File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
h, m = map(int, input().split())
plus_minute = int(input())

add_hour = 0

new_m = 0
new_h = 0

# 분 계산
if m + plus_minute > 59:
    add_hour = ((m + plus_minute) // 60)
    new_m = (m + plus_minute) % 60
else:
    new_m += plus_minute + m

# 시간 계산
if h + add_hour > 23:
    new_h = h + add_hour - 24
else:
    new_h = h + add_hour

print(new_h, new_m)

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().split("\n");

let [h, m] = input[0].split(" ").map(Number);
let plusMinute = Number(input[1]);

let totalMinute = h * 60 + m + plusMinute;

h = parseInt(totalMinute / 60);
m = parseInt(totalMinute % 60);
if(h > 23){
  h -= 24;
}


console.log(`${h} ${m}`);